src/util/convertColorsToPandaCSS.ts

```diff
@@ -1,7 +1,7 @@
 import { SemanticTokens, Tokens } from '@pandacss/dev'
-import { colors } from '../selectors/themeColors'
+import colors from '../colors.config'
 
-/** Converts theme colors to PandaCSS tokens (function will be removed after the conversion to PandaCSS is complete). */
```

Comment:
Do we now plan to keep this function? I am curious why we must keep it, if we originally expected to remove it.

Reply:
It's because we sometimes still need the old colors config, to use when a raw color value is needed, such as for passing in `rgbToHex` `const textHexColor = color ? addAlphaToHex(rgbToHex(themeColor[color])) : undefined` (this is from ColorPicker.tsx)